morpheus/_lib/include/morpheus/stages/preprocess_nlp.hpp

```diff
@@ -86,6 +88,7 @@ struct PreprocessNLPStageInterfaceProxy
                                                                           bool truncation,
                                                                           bool do_lower_case,
                                                                           bool add_special_token,
+                                                                          const std::string column,
```

Comment:
I would make this a 
```suggestion
                                                                          const std::string& column,
```

Reply:
This is more than a nitpick. Good catch. We should either be using pass-by-value and move semantics, or const-ref.